lib/EditableList/EditableListForm.js

```diff
@@ -43,6 +43,10 @@ const propTypes = {
    * Label for the 'Add' button
    */
   createButtonLabel: PropTypes.node,
+  /**
+   *  boolean that indicates if the component is editable (has 'New' button and column 'actions').
+   */
+  editable: PropTypes.bool,
```

Comment:
This should probably be added as `true` to `defaultProps` to preserve functionality in instances of `<EditableList>` without `<ControlledVocab>`.